tests/testthat/test-get_age_model.R

```diff
@@ -7,4 +7,6 @@ test_that("get_age_model works", {
                                            column_id = "test"),
                           warn_args = list(column_id = -503),
                           col_no = 15)
+  # No args provided
+  expect_error(get_age_model())
```

Comment:
You can actually test this by adding an empty list to the `fail_args` argument.

Reply:
Doh! Of course - updated!